lib/origen_testers/flow.rb

```diff
@@ -47,7 +47,35 @@ def self.unique_ids=(val)
     def lines
       @lines
     end
+    
+    def self.ht_comments
```

Comment:
Can you add the following comment to all of these methods please:

~~~ruby
# @api private
def self.ht_comments
~~~

That indicates that even although these are not private methods, they are not intended to be part of the user API and they will be marked as such in the generated API docs.

Here's an example from an existing method that I picked at random - http://origen-sdk.org/testers/api/OrigenTesters/VectorGenerator.html#remove_store_from_vector-instance_method